onnxruntime/python/tools/transformers/optimizer.py

```diff
@@ -248,7 +250,7 @@ def optimize_model(
             else [
                 "MatMulScaleFusion",
                 "MatMulAddFusion",
-                "SimplifiedLayerNormFusion",
```

Comment:
"SimplifiedLayerNormFusion" might impact other model type?

Reply:
I think at the moment it's no harm to remove SimplifiedLayerNormFusion